fmt/printf.ts

```diff
@@ -453,7 +453,7 @@ class Printf {
   /** Handle verb */
   handleVerb() {
     const verb = this.format[this.i];
-    this.verb = verb;
+    this.verb = verb || this.verb;
```

Comment:
Can you please explain this change?

Reply:
Because the  `verb` maybe undefined, So I think this can reusing the `this.verb`, which is initialized to the empty string.